### DISCUSSION TITLE
Is there a --flag support in the dotnet tool

### DISCUSSION THREAD

**Discussion by mattleibow** [cake-build/cake#3886] [Q&A] [answered]
Hi folks, love the tool but I am suuuper lazy.

Who wants to type `--pack=true` when I can do `--pack` and it just works. Is this something that I can do or is it more key-value.

> Not gonna mention names but someone on our team really is a pro dev and knows if it can be done, then a machine can be trained to repeat it. Unfortunately, he or she settled on a `--pack=t` which the OCD types like me can do `--pack=true`, but... OCD...

Help!

**Comment by pascalberger** [comment, marked as answer]
You can use the [HasArgument alias](https://cakebuild.net/api/Cake.Common/ArgumentAliases/20DF3649) to check if a specific argument has been passed:

```
if (HasArgument("pack"))
{
    Information("Pack has been specified");
}
```
